packages/applications/scheduled-tasks/src/exécuter-tâches-planifiées.ts

```diff
@@ -29,16 +37,66 @@ import { InvalidOperationError } from '@potentiel-domain/core';
     );
 
     try {
-      switch (tâche.typeTâchePlanifiée.type) {
+      const { identifiantProjet, typeTâchePlanifiée } = tâche;
+      switch (typeTâchePlanifiée.type) {
         case 'garanties-financières.échoir':
-          mediator.send<GarantiesFinancières.ÉchoirGarantiesFinancièresUseCase>({
+          await mediator.send<GarantiesFinancières.ÉchoirGarantiesFinancièresUseCase>({
             type: 'Lauréat.GarantiesFinancières.UseCase.ÉchoirGarantiesFinancières',
             data: {
-              identifiantProjetValue: tâche.identifiantProjet.formatter(),
+              identifiantProjetValue: identifiantProjet.formatter(),
               échuLeValue: DateTime.now().formatter(),
               dateÉchéanceValue: tâche.àExécuterLe.ajouterNombreDeJours(-1).formatter(),
             },
           });
+          break;
+        case 'garanties-financières.rappel-échéance-un-mois':
+        case 'garanties-financières.rappel-échéance-deux-mois':
+          const {
+            nom,
+            localité: { département },
+          } = await mediator.send<ConsulterCandidatureQuery>({
+            type: 'Candidature.Query.ConsulterCandidature',
+            data: {
+              identifiantProjet: identifiantProjet.formatter(),
+            },
+          });
+
+          const porteurs = await récupérerPorteursParIdentifiantProjetAdapter(
+            tâche.identifiantProjet,
+          );
+          const dreals = await récupérerDrealsParIdentifiantProjetAdapter(identifiantProjet);
+          const nombreDeMois = tâche.typeTâchePlanifiée.estÉgaleÀ(
+            TypeTâchePlanifiée.garantiesFinancieresRappelÉchéanceUnMois,
+          )
+            ? '1'
+            : '2';
+
+          const { BASE_URL } = process.env;
+
+          await sendEmail({
+            messageSubject: `Les garanties financières pour le projet ${nom} arrivent à échéance dans ${nombreDeMois} mois`,
```

Comment:
Tu peux utiliser le style qu'on utilise normalement
```    
messageSubject: `Potentiel - Demande d'abandon ${statut} pour le projet ${nomProjet} (${appelOffre} période ${période})`,
```

```suggestion
            messageSubject: `Potentiel - Arrivée à échéance des garanties financières pour le projet ${nom} dans ${nombreDeMois} mois`,
```